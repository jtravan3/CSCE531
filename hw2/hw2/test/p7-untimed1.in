a = 3
b = 5
(#(a-2)-1)*3
#(1+1)
#a
2+#(#a%4)
#b
b = ##a-1
a = #b
11
12
10
############10
############11
############12
