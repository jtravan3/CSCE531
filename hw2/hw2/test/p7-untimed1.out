Welcome to the Simple Expression Evaluator.
Enter one expression per line, end with ^D

   1: 3
   2: 5
   3: 6
   4: 5
   5: 6
   6: 7
   7: 6
   8: 6
   9: 7
  10: 11
  11: 12
  12: 10
  13: 10
  14: 11
  15: 12
  16: 
